### PR TITLE
Create react components that hold state and render

### PR DESCRIPTION
**Pull request by KimarieHB**
basic bonuses text, create components file, add jquery to package.json

### CODE REVIEW

File: client/src/index.jsx [hunk 1]
```diff
@@ -1,15 +1,37 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import $ from 'jquery';
+import Bonuses from './components/bonuses.jsx';
```
Comment (gektin) [inline client/src/index.jsx:4]: We should confirm whether it is conventional to capitalize the names of component files if they are being imported with capital letters.

File: client/src/index.jsx [hunk 2]
```diff
@@ -1,15 +1,37 @@
...
+
 
 class App extends React.Component {
   constructor(props) {
     super(props);
+    this.state = {
+      bundleNumber: 1, //number
```
Comment (gektin) [inline client/src/index.jsx:11]: How are you planning to confirm that this is a number when it isn't hard coded? And where will errors related to this value be handled?

File: client/src/index.jsx [hunk 3]
```diff
@@ -1,15 +1,37 @@
...
+      bonus: null, //object
+      selectedSong: '', //url
```
Comment (gektin) [inline client/src/index.jsx:13]: When your other two commented out indicators tell you the data type it's expecting, `//url` probably should be something like, `//string, representing url`.

File: client/src/index.jsx [hunk 4]
```diff
@@ -1,15 +1,37 @@
...
+    };
+
+    this.selectSong = this.selectSong.bind(this);
+  }
+  componentDidMount() {
+    $.get(`/bonus/${this.state.bundleNumber}`, (data) => {
+      this.setState({ bonus: data });
+    });
```
Comment (gektin) [inline client/src/index.jsx:21]: Consider handling errorful returned data somehow here. If you get a misshapen or non- object back for your `this.state.bonus`, you'll be passing through a faulty prop to your render of the Bonuses component.

File: database/index.js [hunk 5]
```diff
@@ -14,7 +14,7 @@ mongoose.connect('mongodb://localhost/bonuses', {
 
 // Get bonus associated with bundle
 const getBonuses = (bundleId, callback) => {
-  Soundtrack.find({ bundle_id: bundleId }, (err, data) => {
+  Soundtrack.find({ bundle_id: bundleId }, { _id: 0, 'bonus_info._id': 0, 'bonus_info.tracklist._id': 0 }, (err, data) => {
```
Comment (gektin) [inline database/index.js:17]: What's the purpose of this added object here, and the extended objects on lines 29 and 41?

**Review by gektin — APPROVED** [review]
A few minor thoughts and a question in line. Otherwise, I thought this looked good and made sense as a direction to take it. There were a couple of places in client/src/index.jsx where there was unused code, but it looks like that will be used in forthcoming PRs soon. Thus, I've left out any such comments in line.  One other thing is to consider whether you should be checking your bundle.js in every time or adding it your .gitignore and writing a build script.

File: client/src/index.jsx (hunk 1 above)
Comment (KimarieHB) [reply client/src/index.jsx:4]: Okay.

File: client/src/index.jsx (hunk 2 above)
Comment (KimarieHB) [reply client/src/index.jsx:11]: I am rendering basic data as text first and then will write everything else

File: client/src/index.jsx (hunk 3 above)
Comment (KimarieHB) [reply client/src/index.jsx:13]: okay

File: client/src/index.jsx (hunk 4 above)
Comment (KimarieHB) [reply client/src/index.jsx:21]: I will keep that in mind as I move forward.

File: database/index.js (hunk 5 above)
Comment (KimarieHB) [reply database/index.js:17]: Mongo adds it's automated mega `_id` to every object as it is added and I hate looking at it and don't plan on referencing it. The added object is a query filter that takes out all of the MongoDB `_id`s before returning the query results.

**Pull request merged**